webctrl/script/api_webctrl.py

```diff
@@ -116,12 +117,15 @@ def insert_readings_into_database(conn, readings, sensor):
             #'d' type values stand for digital; are like booleans
             elif key is 'a' or key is 'd':
                 reading_value = reading[key]
-        # subtract 10 hours from reading time for comparison because it's GMT and last_updated_datetime is GMT - 10
-        if reading_time.subtract(hours=10) > pendulum.instance(sensor.last_updated_datetime):
+        # subtract 10 hours from reading time for comparison because it's GMT and last_updated_datetime is GMT - 10 and
+        # check that current reading time is not a duplicate of previous reading time before inserting reading
+        if reading_time.subtract(hours=10) > pendulum.instance(sensor.last_updated_datetime) and reading_time != previous_reading_time:
             reading_row = orm_webctrl.Reading(purpose_id=sensor.purpose_id, datetime=reading_time, reading=reading_value, units=sensor.unit, upload_timestamp=current_time)
             conn.add(reading_row)
             rows_inserted += 1
             new_last_updated_datetime = reading_time
+            previous_reading_time = reading_time
```

Comment:
@carlosparadis What if I add a comment above this line of code like "Set variable to confirm next reading is not a duplicate, before inserting next reading"